### PR TITLE
elm-format 0.2.0 (new formula)

### PR DESCRIPTION
**Pull request by enilsen16**

### CODE REVIEW

File: Library/Formula/elm-format.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+require "language/haskell"
+
+class ElmFormat < Formula
+  include Language::Haskell::Cabal
+  desc "formats Elm source code based on the official Elm Style Guide"
+  homepage "https://github.com/avh4/elm-format"
+  url "https://github.com/avh4/elm-format/archive/0.2.0-alpha.tar.gz"
+  version "3"
```
Comment (bfontaine) [inline Library/Formula/elm-format.rb]: Why is it version `3` if upstream tags it `0.2.0-alpha`?

File: Library/Formula/elm-format.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+  test do
+    src_path = testpath/"Hello.elm"
+    src_path.write <<-EOS.undent
+      import Html exposing (text)
+      main = text "Hello, world!"
+    EOS
+
+    system bin/"elm-format", testpath/"Hello.elm", "--yes"
+    ## assert output is formatted
```
Comment (bfontaine) [inline Library/Formula/elm-format.rb]: You can remove this comment.

File: Library/Formula/elm-format.rb (hunk 1 above)
Comment (enilsen16) [inline Library/Formula/elm-format.rb]: `0.2.0-alpha` is the third release. I was just following that scheme.

https://github.com/avh4/elm-format/releases

Comment (bfontaine) [inline Library/Formula/elm-format.rb]: It seems to be an alpha version; we need a stable release. You can submit this formula to our [`homebrew/devel-only`](https://github.com/Homebrew/homebrew-devel-only) tap instead.

**Comment by enilsen16** [comment]
Please merge https://github.com/Homebrew/homebrew-devel-only/pull/41 instead.

**Pull request closed without merging**